rbac/rbac/middleware.py

```diff
@@ -168,7 +168,11 @@ def process_request(self, request):  # pylint: disable=R1710
             _, json_rh_auth = extract_header(request, self.header)
             user.account = json_rh_auth.get("identity", {})["account_number"]
             user.org_id = json_rh_auth.get("identity", {}).get("org_id")
-            user_info = json_rh_auth.get("identity", {}).get("user", {})
+            usertype = json_rh_auth.get("identity", {}).get("type")
```

Comment:
From my comment above in `dev_middleware.py`, do we _need_ to allow the Turnpike identities to go through this `get_tenant` method? It looks like the way it currently works, we do. Testing locally with an updated Turnpike identity leads to a bunch of issues with this because the account, org, username, etc., don't exist.

I think we could change the access on some of these variables to use `.get()` which wouldn't fail with a `KeyError`, but that would negate the check on `KeyError` below, to return a 401. Also, it wouldn't set the tenant on the request because there's no account. We'll probably need to get that from the internal request?